core-bundle/src/Image/Studio/Figure.php

```diff
@@ -124,6 +125,19 @@ public function getMetadata(): Metadata
         return $this->metadata;
     }
 
+    public function getJsonLd(Request $request = null): array
+    {
+        $schemaBase = $request ? $request->getSchemeAndHttpHost() : '';
+
+        $jsonLd = [
+            '@type' => 'ImageObject',
+            'identifier' => $schemaBase.'#/schema/image/'.$this->getImage()->getImageSrc(), // TODO: replace with UUID once this is available
+            'contentUrl' => $this->getImage()->getImageSrc(),
+        ];
+
+        return array_merge($jsonLd, $this->getMetadata()->getJsonLd('ImageObject'));
```

Comment:
We should check if metadata exists, otherwise there would be an exception:

```suggestion
        if(!$this->hasMetadata()) {
            return $jsonLd;
        }

        return array_merge($jsonLd, $this->getMetadata()->getJsonLd('ImageObject'));
```